analyses/tcga-capture-kit-investigation/scripts/intersect-bed-maf.sh

```diff
@@ -0,0 +1,52 @@
+#!/bin/bash
+##############################
+# 
+# This script intersects all the BED and then use that to intersect with 
+# released PBTA and TCGA MAF and then counted all the mutation number within 
+# that intersection region, and then mapped that counts to the project and 
+# tumor type for each sample.
+# 
+##############################
+
+
+## set data path variable 
+DATA='../../data'
+SCRATCH='../../scratch'
+
+
+## get intersection between all the BED files
+bedtools intersect -a $DATA/WGS.hg38.strelka2.unpadded.bed -b $DATA/WGS.hg38.mutect2.vardict.unpadded.bed \
+| bedtools intersect -a - -b $DATA/WGS.hg38.lancet.unpadded.bed \
+| bedtools intersect -a - -b results/whole_exome_agilent_1.1_refseq_plus_3_boosters.targetIntervals.Gh38.bed \
+| bedtools intersect -a - -b results/whole_exome_agilent_designed_120.targetIntervals.Gh38.bed \
+| bedtools intersect -a - -b results/whole_exome_agilent_plus_tcga_6k.targetIntervals.Gh38.bed \
+| bedtools intersect -a - -b results/tcga_6k_genes.targetIntervals.Gh38.bed \
```

Comment:
I realized that this list of files is not the same as the ones in the results directory after CrossMap addition.

This list should probably be autogenerated from `results/*.targetIntervals.Crossmapped_to_Gh38.bed`.

Reply:
While this is now correct, I think it might still be useful to have this be a bit more automated in case added files TCGA files were to be added. I think the easiest way to do this would be to make a subdirectory of `results/` that holds only the bed files, which may make later processing easier as well.